secrets/values.yaml

```diff
@@ -0,0 +1,90 @@
+# secret main file
+
+# global values
+global:
+  auth: "oidc"
+  authIdToken: "FIXME"
+  authVO: "Rubin.production"
+
+  activemq:
+    serverList:
+      - "activemq-dev:61613"
+
+    basePasswd: "FIXME"
+
+  idds:
+    restHost: "idds-rest"
+    restPort: 8443
+
+  rucio:
+    rucioHost: "sprucio01.sdcc.bnl.gov:443"
+    authHost: "sprucio01.sdcc.bnl.gov:443"
+    username: "iddsv1"
+    password: "FIXME"
+    account: "iddsv1"
+
+  oidc:
```

Comment:
This is not flexible to adding a new VO. How about changing this to a list? e.g.

```
oidc:
  audience: "..."
  config_url: "..."
  clients:
    - name: "panda_dev"
      id: "..."
      secret: "..."
   - name: "rubin"
      id: "...
```
?

Then

https://github.com/PanDAWMS/panda-k8s/blob/67059afe8a8d725a31011ba44b98aab885599d4f/secrets/templates/panda.yaml#L54-L57

this can be changed like

```
  {{- range $client := .Values.oidc.clients }}
  {{ $client.name }}_auth_config.json: |-
    {{ $.Files.Get "files/auth_config.json.template" | replace "ID" $client.id | replace "NAME" ( split "." $client.secret )._0 | replace "SECRET" $client.secret | replace "CONF" $oidc.config_url | replace "AUDIENCE" $oidc.audience }}
  {{ end }}
```

files/auth_config.json.template could be something like
```
{"client_secret": "SECRET", "audience": "AUDIENCE", "client_id": "ID", "oidc_config_url": "CONF", "vo": "NAME"}
```

Reply:
Good idea. Will update it later.

Reply:
solved